ISISComputingGroup/ibex_gui
# Ticket 6551: Error if concentration too high

### Description of work

See https://github.com/ISISComputingGroup/IBEX/issues/6551, https://github.com/ISISComputingGroup/IBEX/issues/5093, https://github.com/ISISComputingGroup/IBEX/issues/4681, https://github.com/ISISComputingGroup/IBEX/issues/4404

To test:

* Run the jasco in the IOC test framework
* Confirm setting concentrations over 100% stops the user from start/stopping a run and gives adequate warnings
* Confirm requirements of other linked tickets are met


---

#### Code Review


### Final Steps
